pkg/controllers/overrider/common_test.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/google/go-cmp/cmp"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
```suggestion
	corev1 "k8s.io/api/core/v1"
```

Reply:
I wonder why it's better to name it corev1?

Reply:
v1 is quite common as the k8 api package. 

Without looking at the actual import path, we don't know what's the package. corev1 is quite commonly used in our repo now.
So it's easier to figure out it stands for "k8s.io/api/core/v1"